autoload/vista/renderer.vim

```diff
@@ -13,10 +13,12 @@ let s:icons = {
 \    'variables': "\uf71b",
 \    'const': "\uf8ff",
 \    'constant': "\uf8ff",
+\    'constructor': "\uf976",
 \    'method': "\uf6a6",
 \    'package': "\ue612",
 \    'packages': "\ue612",
 \    'enum': "\uf702",
+\    'enummember': "e",
```

Comment:
```suggestion
\    'enummember': "\uf702",
```

Use `"\uf702"` or another icon, just `e` is not good IMO.

Reply:
I figure that `enummember`s will almost always be scoped inside an `enum`, so it would make sense if we're using a fancy capital `E` for the enum itself we could use lowercase `e` for its members